admin-dev/themes/default/js/bundle/module/module.js

```diff
@@ -554,13 +542,11 @@ var AdminModuleController = function () {
                 var actionMenuObj = data.actionMenuObj;
                 var moduleTechName = data.techName;
 
-                //actionMenuObj.fadeOut();
-
                 var urlActionSegment = bulkActionToUrl[requestedBulkAction];
-                var actionUrlBase = baseActionUrl.replace('@', urlActionSegment);
+                baseActionUrl.replace('@', urlActionSegment);
```

Comment:
You are right, you can totally remove this as of ```module_card.js``` introduction - since its this one which effectively do the ajax call :)